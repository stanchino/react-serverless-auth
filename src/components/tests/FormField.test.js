import React from "react";
import renderer from "react-test-renderer";

import { FormField } from "..";

describe("FormField", () => {
    it("matches the snapshot", () => {
        const tree = renderer.create(<FormField meta={ { touched: true, error: "error" } }/>).toJSON();
        expect(tree).toMatchSnapshot();
    });
});