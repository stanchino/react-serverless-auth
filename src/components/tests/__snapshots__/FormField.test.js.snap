// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormField matches the snapshot 1`] = `
<div>
  <label />
  <div>
    <input
      placeholder={undefined}
      type={undefined}
    />
    <span>
      error
    </span>
  </div>
</div>
`;
